HW6(Async-RegExp)/HW6_Trupkina.js

```diff
@@ -0,0 +1,45 @@
+
+function validEmail() {
+    //Set the default value
+    let email = prompt('Type correctly email please', `example@gmail.com`);
+    const regexp_email = /^([\w\.+-]{3,})+@[A-z0-9-]{3,10}\.[A-z]{2,6}$/;//we can't use \w in domen name and sub name
+    if (email === null) return undefined;//i catch cancellation
+    else if (regexp_email.test(email) === true) return alert('This is your Email:\n' + email);
```

Comment:
pls, try do not use syntax like that

for readable better write

``
if (...) {
  ...
} else if (...) {
  ...
}
``